wye/templates/workshops/workshop_detail.html

```diff
@@ -51,7 +51,11 @@ <h1 class="no-space">Workshop Details</h1>
                     <li class="list-group-item">
                         <div class="row">
                             <div class="col-xs-4"><strong>Instructors</strong></div>
```

Comment:
The profile page presently just show no of workshop conducted and other pythonexpress stats.
Probably with this change we need to enhance profile page fields displays.(Image)
The intent is to display tutor profile for college poc to understand about tutor .

![screen shot 2016-10-09 at 4 40 13 am](https://cloud.githubusercontent.com/assets/316253/19216482/0bef87c2-8ddb-11e6-847f-44ddeaa5fa5c.png)